app/src/main/java/com/sun_asterisk/weather_03/screen/home/HomePresenter.java

```diff
@@ -0,0 +1,10 @@
+package com.sun_asterisk.weather_03.screen.home;
+
+public class HomePresenter<V> implements HomeContract.Presenter {
+    private HomeContract.View mView;
+
+    @Override
+    public void setView(Object view) {
+        mView = (HomeContract.View) view;
```

Comment:
default nó luôn là View nên không cần phải ép kiểu e
```suggestion
        mView =  view;
```